policy/predicate/signature.go

```diff
@@ -27,26 +27,40 @@ type HasValidSignatures bool
 
 var _ Predicate = HasValidSignatures(false)
 
-func (pred HasValidSignatures) Evaluate(ctx context.Context, prctx pull.Context) (bool, string, error) {
+func (pred HasValidSignatures) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
 	commits, err := prctx.Commits()
+
+	predicateResult := common.PredicateResult{
+		Satisfied:       false,
+		ConditionPhrase: "have valid signatures",
```

Comment:
```suggestion
		ValuePhrase: "commits",
		ConditionPhrase: "have valid signatures",
```

This is one of the few cases where both `ConditionMap` and `ConditionValues` are empty. We'll want to either account for that in the template or change this to set one of them:

```go
ValuePhrase: "commits",
ConditionPhrase: "have",
ConditionValues: []string{"valid signatures"},
```